packages/block-library/src/social-link/index.php

```diff
@@ -19,7 +19,12 @@ function render_block_core_social_link( $attributes, $content, $block ) {
 
 	$service    = ( isset( $attributes['service'] ) ) ? $attributes['service'] : 'Icon';
 	$url        = ( isset( $attributes['url'] ) ) ? $attributes['url'] : false;
-	$label      = ( isset( $attributes['label'] ) ) ? $attributes['label'] : block_core_social_link_get_name( $service );
+	$label      = ( isset( $attributes['label'] ) ) ? $attributes['label'] : sprintf(
+		/* translators: %1$s: Social-network name. %2$s: URL. */
+		__( '%1$s: %2$s', 'gutenberg' ),
```

Comment:
Does it really need to be applied in the translation? It looks like `sprintf` is enough.

Reply:
If it's not in the translation, then how would an RTL language reverse it, or how would French add a space _before_ the `:`? 🤔

Reply:
I see now it's only used with `aria-label` so it's going to be read by the assistive technology which is yet another set of challenges.